app/src/main/java/com/github/se/eduverse/ui/setting/Setting.kt

```diff
@@ -105,7 +107,7 @@ fun SettingsScreen(navigationActions: NavigationActions) {
               label = "Language",
               selectedOption = selectedLanguage,
               options = listOf("Français", "English"),
```

Comment:
I think it's better to leave it like this, in order to not forget that it could be a possible future implementation. If we decide to not implement that I will make sure to delete this part.